api/types/types.go

```diff
@@ -291,10 +287,13 @@ type Credential struct {
 	Hosts       string
 	ConnectJson bool
 	Post        bool
+	SiteOwner   bool
+	Data        map[string][]byte
 }
 
 type CertAuthority struct {
-	Name string
+	Name      string
+	SiteOwner bool
 }
```

Comment:
What is the SiteOwner for?

Reply:
I added this to distinguish owner ref for all resources being either the site or the router deployment as creating the "spec" van_router_create is separate from the resource instantiations. There may be a better name to use or alternative approach.

Reply:
I think all that matters is that the root owner is the configmap (so that deleting that delets everything). If some resources are 'owned' by the router deployment, which is itself 'owned' by the configmap, that is fine. I don't think there is a firm reason to choose one over the other (its more of a pragmatic choice as things evolve) so I would argue it shouldn't muddy the data structures. Wdyt?

Reply:
Totally agree, will simplify with single root owner.